src/components/html/EcApplication.html

```diff
@@ -1,41 +1,66 @@
-<div class="container py-4">
-  <div class="row">
-    <div class="col">
-      <a-card hoverable style="width: 100%">
-        <template class="ant-card-actions" slot="actions">
-          <a-icon type="api" title="Instalar agora" @click="installApp" v-if="!isInstalled" class="ec-application__actions"/>
-          <a-icon type="setting" title="Configurar aplicativo" @click="toggleSettings" v-if="isInstalled" class="ec-application__actions"/>
-          <a-popconfirm
-            title="Tem certeza que deseja excluir o aplicativo?"
-            @confirm="deleteApp"
-            okText="Sim"
-            cancelText="Não"
-            v-if="isInstalled"
-          >
-            <a-icon type="delete" title="Excluir aplicativo" class="ec-application__actions"/>
-          </a-popconfirm>
-        </template>
-        <a-card-meta :title="title" :description="shortDescription">
-          <a-avatar
-            slot="avatar"
-            :src="icon"
-          />
-        </a-card-meta>
-      </a-card>
+<div class="ec-application">
+  <a-row :gutter="24">
+    <div v-if="loading">
```

Comment:
Na verdade você deveria mostrar o conteúdo se _não_ estiver carregando...

Reply:
Ou isto era um `isLoaded` em vez de `loading`?

Reply:
Se era um `isLoaded` eu preciso que você altere o nome da variável pra ser condizente (`isLoaded`, `loaded`, `isReady`...), e aí na verdade acho que o correto seria setar o `this.isLoaded = true` no `localApplication.set`

Reply:
Eu alterei o nome.

Reply:
acho que ja tinha essa variavel no component e eu só continuei usando.

Reply:
provável que sim, mas se era `loading` é porque indicaria o momento em que o dados estavam sendo carregados, se você resolveu implementar de outra forma o nome da variável tem muda...

Reply:
você solicitou o review mas sem alteração ainda, talvez não fez o push aí ?

Reply:
deu erro com minha autenticação e eu nem percebi que nao foi 🤪